src/util.h

```diff
@@ -117,6 +117,20 @@ namespace rhost {
             }
         };
 
+        struct blob {
```

Comment:
Also, consider how this will be used (if it will be) for receiving messages with blobs. We'll probably need a separate type there that would actually hold data / manage lifetime.